tools/lxdclient/config.go

```diff
@@ -60,13 +60,24 @@ func (cfg Config) UsingTCPRemote() (Config, error) {
 		return cfg, nil
 	}
 
+	/* Here, we rely on the fact that Connect() does a ServerStatus.
```

Comment:
Connect() doesn't do a ServerStatus anymore for HTTP connections. Shouldn't you use Finger() if you really want it to trigger it?